backend/timed/tracking/signals.py

```diff
@@ -15,6 +16,12 @@ def update_rejected_on_reports(sender, instance, **kwargs):  # noqa: ARG001
             instance.rejected = False
 
 
+@receiver(post_save, sender=Report)
+def update_search_vector(sender, instance, **kwargs):  # noqa: ARG001
+    """Update comment search vector."""
+    instance.search_vector = SearchVector("comment", config="english")
```

Comment:
This will not have any effect: The `post_save` is triggered after the object is saved to DB, and you would need to save it again, causing an infinite loop.

If you just change this to `pre_save`, it will work as intended.

I think it'd be a good idea to write a small test for this:

* Create a report or two
* Trigger the search endpoint and see if it can be found

Reply:
setting it to `pre_save` runs into 

```
Failed to insert expression "SearchVector(...)" . F() expressions can only be used to update, not to insert.
```

Reply:
> I think it'd be a good idea to write a small test for this

https://github.com/adfinis/timed/blob/c1b5588c975fd404dcd25ab44632efac80eef8bd/backend/timed/tracking/tests/test_report.py#L2099-L2122

such a test already exists :)

Reply:
`SearchVector(Value(instance.comment))` should work

Reply:
> such a test already exists :)

Good thing you pointed this out: The filter is still naively doing `comment__search` lookup, completely ignoring the improvements here.

You'll need to update the filter to do something like this:

```python
    def filter_comment(self, queryset, name, value):
        return queryset.filter(
            content_vector=SearchQuery(
                value,
                config="simple", # maybe use "english"?
                search_type="phrase",
            )
        )
```

Reply:
it feels a lot faster now